src/sentry/utils/snuba.py

```diff
@@ -692,7 +693,6 @@ def raw_snql_query(
     # XXX (evanh): This function does none of the extra processing that the
     # other functions do here. It does not add any automatic conditions, format
     # results, nothing. Use at your own risk.
-    metrics.incr("snql.sdk.api", tags={"referrer": referrer or "unknown"})
```

Comment:
See my comment on why this call should not be moved. Please don't put the metric inside the `_apply_cache...` function.

Reply:
Sounds good, thanks for the context on the metric 👍